src/components/layout/MainNavigation.js

```diff
@@ -3,15 +3,22 @@ import NewPostPage from "../../pages/NewPost";
 import classes from "./MainNavigation.module.css";
 import Post from "../UI/Post";
 import Backdrop from "../UI/Backdrop";
+import ErrorModal from "../UI/ErrorModal";
 
-function MainNavigation({setRefreshPost}) {
+function MainNavigation({ setRefreshPost }) {
   const [showForm, setShowForm] = useState(false);
+  const [showError, setShowError] = useState(false);
```

Comment:
You might be able to check whether the `error` text is null or empty rather than having to introduce a separate boolean state variable

Reply:
yeah that's true